WMIAdventure/frontend/src/js/components/battle/atoms/BuffPreviewBlock/styled-components/Container.js

```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components';
+
+function colorHandle(theme, type) {
+    switch (type) {
+        case null:
+            return theme.colors.red;
+        case 1:
+            return theme.colors.red;
+        case 2:
+            return theme.colors.purplyPinky;
+        case 5:
+            return theme.colors.yellowyOrangy;
+        case 6:
+            return theme.colors.greenyBluey;

```

Comment:
Co oznaczają te numerki? Dobrze byłoby jakby stały za jakimiś zmiennymi które powiedzą co to jest